clab/docker.go

```diff
@@ -119,6 +119,18 @@ func (c *cLab) CreateBridge(ctx context.Context) (err error) {
 func (c *cLab) DeleteBridge(ctx context.Context) (err error) {
 	nctx, cancel := context.WithTimeout(ctx, c.timeout)
 	defer cancel()
+	nres, err := c.DockerClient.NetworkInspect(ctx, c.Conf.DockerInfo.Bridge)
+	if err != nil {
+		return err
+	}
+	numEndpoints := len(nres.Containers)
+	if numEndpoints > 0 {
+		log.Warnf("network '%s' has %d active endpoints", c.Conf.DockerInfo.Bridge, numEndpoints)
+		for _, endp := range nres.Containers {
```

Comment:
do we want to enter this loop only if the debug is set?

Reply:
we could, but I don't see much gain